package.json

```diff
@@ -35,15 +35,20 @@
     "babel-eslint": "8.0.2",
     "babel-jest": "21.2.0",
     "babel-loader": "7.1.2",
+    "babel-plugin-syntax-async-functions": "^6.13.0",
```

Comment:
Every dependency is tha package.json file should be an exact version (without the ^), so that the version is fixed and greenkeeper can open a pull request for us when one of our dependency released a new version on npm.